src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -6549,8 +6552,13 @@ namespace CondenserLoopTowers {
             NumTransferUnits = UAactual / CapacityRatioMin;
             // calculate heat exchanger effectiveness
             if (CapacityRatio <= 0.995) {
-                effectiveness = (1.0 - std::exp(-1.0 * NumTransferUnits * (1.0 - CapacityRatio))) /
-                                (1.0 - CapacityRatio * std::exp(-1.0 * NumTransferUnits * (1.0 - CapacityRatio)));
+                Real64 Exponent = NumTransferUnits * (1.0 - CapacityRatio);
+                if (Exponent >= 700.0) {
```

Comment:
This 700 is documented in the engineering reference and/or elsewhere, right?

Reply:
Also, can you reuse this evaluation?  Like actually use the exponent variable in the expression?  If you needed you could just set `Real64 Exponent = negative value` and then change the logic to `if (Exponent <= -700)` and then use the value directly inside `std::exp`.

Reply:
The 700 is not documented in the eng. ref. This value is used else where in the code to avoid underflow/overflow.  I think there is an issue for this limit #7233 and addressed in PR #7414. Now, @lgu1234 proposed 70.0 as the limit instead of 700.0.

Reply:
Modified the code to use the Exponent local variable directly in the equations.